bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/LGWebOSHandlerFactory.java

```diff
@@ -97,6 +97,7 @@ protected void deactivate(ComponentContext componentContext) {
         super.deactivate(componentContext);
         try {
             this.webSocketClient.stop();
+            this.webSocketClient.destroy();
```

Comment:
Destroy means this  instance in webSocketClient cannot be started again.
Since we create this instance in the constructor it means this is OK as long as activate is never called again after deactivate.
If this is assured by the OSGI specification, then it will work. I am not an expert on this OSGI lifecycle.
the diagram here suggests that it is possible to run through a cycle of deactivate and activate:
https://www.openhab.org/docs/developer/osgi/osgids.html#component-lifecycle

Reply:
Ok, I will remove the destroy and check again the 12 binfings with this new information.

Reply:
Removed